portfolio/src/main/webapp/images/projImages/info

```diff
@@ -0,0 +1 @@
+This folder contains images to be used in the project page
```

Comment:
You may want to consider using all lowercase for file/folder names. Case sensitivity in filenames is not consistent across different OSes and server frameworks, and it's an easy way to spend hours trying to figure out why a file won't load (in my experience at least).